### DISCUSSION TITLE
Activate additional fields that only the recipient sees

### DISCUSSION THREAD

**Discussion by cesar-sanjuan** [globaleaks/globaleaks-whistleblowing-software#3431] [General]
Is it possible to define and activate fields or questions that are only available to the recipient?
That is, they do not appear when the complainant is filing the complaint and they are visible and editable by the recipient. And that if the recipient uses them they are not visible to the complainant
thank you!!!

**Comment by evilaliv3** [comment]
Thank you for your questions.

At the moment the software do not have such a features.

On the ticketing system we have collected some suggestions to make this or similar features but there is currently no active development on the specific aspect. 

It would be helpful if you can give at a look to it and annotate your needs that would be helpful to developers and funders.
